### PR TITLE
Add documentation to integrate Dash/Alfred. #213

### PR DESCRIPTION
**Pull request by hofmannsven**

### CODE REVIEW

File: installing.md [hunk 1]
```diff
@@ -88,6 +88,22 @@ Note: the Oh My Zsh plugin comes with the bash completion script included, so it
 
 To have this change take effect in your currently active shell, run `source ~/.zshrc` afterwards.
 
+### Dash/Alfred workflow
+
+If you're using _Dash_ and _Alfred_, you can look up WP-CLI command information.
```
Comment (schlessera) [inline installing.md]:
```suggestion
If you're using _Dash_ and _Alfred_, you can add a custom Alfred workflow to look up WP-CLI command information.
```

File: installing.md [hunk 2]
```diff
@@ -88,6 +88,22 @@ Note: the Oh My Zsh plugin comes with the bash completion script included, so it
...
+ 
+Open Dash and download the [WP-CLI docset](https://github.com/wp-cli/dash-docset-generator):
+Dash › Preferences › Downloads › User Contributions › Search for WP-CLI
+
+Still in Dash, activate the Alfred integration:
+Dash › Preferences › Integration › Alfred
+
+#### Usage
+
+Open Alfred and search for: `wp-cli plugin`
```
Comment (schlessera) [inline installing.md]:
```suggestion
Open Alfred and try searching for a specific command. For example, this gives you an overview of the `plugin` command and its subcommands: `wp-cli plugin`
```

**Comment by schlessera** [comment]
Thanks for the PR, @hofmannsven !

**Comment by holtjohnson** [comment]
Thanks for this!

**Pull request merged**